vulcanai/models/metrics.py

```diff
@@ -892,12 +893,41 @@ def _boot_cv(network, data_loader, k, epochs, retain_graph, valid_interv,
             logger.info(
                 "\n\n***KeyboardInterrupt: Cross validate stopped \
                 prematurely.***\n\n")
-	
+
+    @staticmethod
+    def stratified_split(dataset, k, strata_column):
+
```

Comment:
should we add some assertions about the type of these variable? might as well add a tiny but of docs here